Subject: [PATCH] 계정연동 테스트
---
Index: app/src/main/java/com/example/ph_k/MyPageActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/ph_k/MyPageActivity.java b/app/src/main/java/com/example/ph_k/MyPageActivity.java
--- a/app/src/main/java/com/example/ph_k/MyPageActivity.java	(revision 87d948d0fa520e99944d42e6ea654e69f19ed885)
+++ b/app/src/main/java/com/example/ph_k/MyPageActivity.java	(revision bdbb5c48a048b2502a3b54d24520bedb366a7924)
@@ -20,8 +20,6 @@
 public class MyPageActivity extends AppCompatActivity {
 
     private DrawerLayout drawerLayout;
-    private NavigationView navigationView;
-    private BottomNavigationView bottomNavigationView;
     private ActionBarDrawerToggle toggle;
 
     @Override
@@ -39,8 +37,8 @@
 
         // 네비게이션과 하단 네비게이션 설정
         drawerLayout = findViewById(R.id.drawerLayout);
-        navigationView = findViewById(R.id.navigationView);
-        bottomNavigationView = findViewById(R.id.bottomNavigationView);
+        NavigationView navigationView = findViewById(R.id.navigationView);
+        BottomNavigationView bottomNavigationView = findViewById(R.id.bottomNavigationView);
         Toolbar toolbar = findViewById(R.id.toolbar);
 
         setSupportActionBar(toolbar);
